contracts/Initializable.sol

```diff
@@ -46,4 +46,6 @@ contract Initializable {
     assembly { cs := extcodesize(address) }
     return cs == 0;
   }
+
+  uint256[50] private ______gap;
```

Comment:
We will be removing Initializable and loading it from zos-lib afterwards, right?

Reply:
Yes - should we add this there?